### PR TITLE
Predefined queries

### PR DESCRIPTION
**Pull request by berggren**
Fixes: #457

### CODE REVIEW

File: timesketch/ui/css/ts.scss [hunk 1]
```diff
@@ -230,11 +230,19 @@ header {
...
 
 .select-view {
     margin-left:5px;
-    outline: none;
+}
+
+.select-graph-view {
+  height:57px;
+  background-color:#f9f9f9;
+  -webkit-border-radius: 0px;
```
Comment (franekp) [inline timesketch/ui/css/ts.scss]: Please write `border-radius` here and use autoprefixer to append prefixes for all browsers.

File: timesketch/ui/css/ts.scss [hunk 2]
```diff
@@ -230,11 +230,19 @@ header {
...
     width:150px;
+    outline: none;
 }
 
 .select-view {
     margin-left:5px;
-    outline: none;
+}
+
+.select-graph-view {
```
Comment (franekp) [inline timesketch/ui/css/ts.scss]: Please be consistent with other css related to graph part, please nest it inside component selector:
```scss
ts-graphs-cypher-query {
    .#{$}__predefined-queries {
        ...
    }
}
```

File: timesketch/ui/graphs/cypher-query.component.html [hunk 3]
```diff
@@ -1,6 +1,14 @@
 <div class="card card-top">
 
   <form class="form-inline" (ngSubmit)="onSubmit()">
+
+    <select class="select select-graph-view pull-left" (change)="onQuerySelect($event.target.value)">
```
Comment (franekp) [inline timesketch/ui/graphs/cypher-query.component.html]: select-graph-view -> ts-graphs-cypher-query__predefined-queries

File: timesketch/ui/css/ts.scss (hunk 1 above)
Comment (franekp) [reply timesketch/ui/css/ts.scss]: https://github.com/postcss/postcss-loader  <- use this

**Comment by franekp** [comment]
Also, use `yarn run lint` or `./run_tests.py --js --no-tests` to lint the code as there are some linter errors in travis build.

File: timesketch/ui/css/ts.scss (hunk 2 above)
Comment (berggren) [reply timesketch/ui/css/ts.scss]: Done.

File: timesketch/ui/css/ts.scss (hunk 1 above)
Comment (berggren) [reply timesketch/ui/css/ts.scss]: Sounds good, will do in a followup CL.

File: timesketch/ui/graphs/cypher-query.component.html (hunk 3 above)
Comment (berggren) [reply timesketch/ui/graphs/cypher-query.component.html]: Done.

**Comment by berggren** [comment]
Done, also linter errors fixed :)
PTAL

**Pull request merged**